src/urllib3/util/retry.py

```diff
@@ -344,6 +345,8 @@ def _is_connection_error(self, err: Exception) -> bool:
         """
         if isinstance(err, ProxyError):
             err = err.original_error
+        elif isinstance(err, NameResolutionError):
+            return True
         return isinstance(err, ConnectTimeoutError)
```

Comment:
Thoughts on `return isinstance(err, (ConnectTimeoutError, NameResolutionError))`?

Reply:
funny how when I see this I felt like it was so obvious I dont get why I did not write it in the first place